configuration_go/schemas/thanos/objstore/objstore.go

```diff
@@ -1,6 +1,6 @@
 package objstore
 
-// Taken from https://github.com/thanos-io/objstore/client/factory.go
+// Taken from github.com/thanos-io/objstore/client/factory.go
```

Comment:
Why these changes? I think we have more changes that code editors identify the link and make it clickable if it's prefixed with `http[s]`.

Reply:
Because the URL does not resolve, you would have to add `/tree/main` and because I am referencing a package here, not a website. So package notation seemed more accurate to me. And it aligns with what is done in `/schemas` of this repo.

Reply:
Maybe we should link it to an specific commit then? Because the code in the linked files can change and this comment then become obsolete and wrong. If we link to the commit it will always be correct (unless a force push happens, of course).

Reply:
fixed